.github/workflows/typespec-validation.yaml

```diff
@@ -0,0 +1,25 @@
+name: Typespec Validation
+
+on: pull_request
+
+jobs:
+  typespec-validation:
+    name: Typespec Validation
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 2
+
+    - name: Setup CI environment
+      uses: ./.github/actions/setup-node-ci
+
+    - name: Validate Impacted Specs
+      run: |
+        ./eng/scripts/TypeSpec-Validation.ps1 -GitClean -Verbose
```

Comment:
We still need to improve the perf of check TSV, when it needs to validate all specs, due to changes in "core files".  I have ideas, but let's do this in a later PR.